src/Fhir.Anonymizer.FunctionalTests/TestResources/patient-basic-target.json

```diff
@@ -1,6 +1,24 @@
 {
   "resourceType": "Patient",
   "id": "example",
+  "meta": {
+    "security": [
+      {
+        "system": "http://terminology.hl7.org/CodeSystem/v3-ObservationValue",
+        "code": "REDACTED",
+        "display": "part of the resource is removed"
+      },
+      {
+        "system": "http://terminology.hl7.org/CodeSystem/v3-ObservationValue",
+        "code": "ABSTRED",
+        "display": "exact value is replaced with a range"
+      },
+      {
+        "code": "PERTURBED",
```

Comment:
Where is "PERTURBED" defined? didn't find it in "https://www.hl7.org/fhir/v3/ObservationValue/cs.html".

Reply:
Yes, it is a label created by our own, as currently there is no available label to describe the behavior of date shift. Details could be found in the [internal discussion](https://microsofthealth.visualstudio.com/Health/_workitems/edit/72972) as well.

Reply:
Thanks Boya for the info!